wraith/bin/wraith.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+read -p "Specify the port number that your local dev environment is running on: " port
+
+port=${port:-5000}
```

Comment:
Just been going through the wraith source code and it looks like it's a YAML config or nothing. Would be nice to specify this stuff with an environment variable but agree that the amount of trickery involved probably doesn't provide as much value. And given our readme recommends `foreman` anyway, this lines up.

So shall we just get rid?